[INFO][0][<module>] Darwin CMPG7MP2G29J2.local 21.3.0 Darwin Kernel Version 21.3.0: Wed Jan  5 21:37:58 PST 2022; root:xnu-8019.80.24~20/RELEASE_ARM64_T6000 x86_64
[INFO][0][<module>] /opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB (2.6)
[INFO][0][<module>] /opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/numpy (1.23.0)
[INFO][0][<module>] /opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/matplotlib (3.5.2)
[INFO][0][<module>] /opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/netCDF4 (1.5.8)
[INFO][0][<module>] /opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/cf_units (3.0.1)
[INFO][0][<module>] /opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/sympy (1.10.1)
[INFO][0][<module>] /opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/mpi4py (3.1.3)
[INFO][0][<module>] 2022-07-08 10:43:59.351695
[DEBUG][0][extractTimeSeries][JULES] Could not find [ra] in the model results
[DEBUG][0][WorkConfront][EcosystemRespiration/FLUXCOM][JULES]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][LPJ-GUESS] Could not find [ra] in the model results
[DEBUG][0][WorkConfront][EcosystemRespiration/FLUXCOM][LPJ-GUESS]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][JULES] Could not find [ra] in the model results
[DEBUG][0][WorkConfront][EcosystemRespiration/FLUXNET2015][JULES]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][LPJ-GUESS] Could not find [ra] in the model results
[DEBUG][0][WorkConfront][EcosystemRespiration/FLUXNET2015][LPJ-GUESS]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][JULES] Could not find [ra] in the model results
[DEBUG][0][WorkConfront][EcosystemRespiration/GBAF][JULES]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][LPJ-GUESS] Could not find [ra] in the model results
[DEBUG][0][WorkConfront][EcosystemRespiration/GBAF][LPJ-GUESS]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][WorkConfront][LeafAreaIndex/AVH15C1][DLEM]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 282, in extractTimeSeries
    var = Variable(filename       = pathName,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Variable.py", line 132, in __init__
    out = il.FromNetCDF4(filename,variable_name,alternate_vars,t0,tf,group=groupname,convert_calendar=convert_calendar)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 790, in FromNetCDF4
    t,t_bnd,cbounds,begin,end,calendar = GetTime(var,t0=t0,tf=tf,convert_calendar=convert_calendar)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 246, in GetTime
    tb = CreateTimeBounds(t,alpha=GuessAlpha(t))
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 149, in GuessAlpha
    t0 = cf.num2date(t[0],t.units,calendar=t.calendar)
  File "src/cftime/_cftime.pyx", line 547, in cftime._cftime.num2date
  File "src/cftime/_cftime.pyx", line 103, in cftime._cftime._dateparse
ValueError: units must be one of 'seconds', 'minutes', 'hours' or 'days' (or singular version of these), got 'years'

[DEBUG][0][extractTimeSeries][JSBACH] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/AVH15C1][JSBACH]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][LPJwsl] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/AVH15C1][LPJwsl]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][ORCHIDEE] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/AVH15C1][ORCHIDEE]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][ORCHIDEE-CNP] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/AVH15C1][ORCHIDEE-CNP]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][WorkConfront][LeafAreaIndex/AVHRR][DLEM]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 282, in extractTimeSeries
    var = Variable(filename       = pathName,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Variable.py", line 132, in __init__
    out = il.FromNetCDF4(filename,variable_name,alternate_vars,t0,tf,group=groupname,convert_calendar=convert_calendar)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 790, in FromNetCDF4
    t,t_bnd,cbounds,begin,end,calendar = GetTime(var,t0=t0,tf=tf,convert_calendar=convert_calendar)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 246, in GetTime
    tb = CreateTimeBounds(t,alpha=GuessAlpha(t))
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 149, in GuessAlpha
    t0 = cf.num2date(t[0],t.units,calendar=t.calendar)
  File "src/cftime/_cftime.pyx", line 547, in cftime._cftime.num2date
  File "src/cftime/_cftime.pyx", line 103, in cftime._cftime._dateparse
ValueError: units must be one of 'seconds', 'minutes', 'hours' or 'days' (or singular version of these), got 'years'

[DEBUG][0][extractTimeSeries][JSBACH] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/AVHRR][JSBACH]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][LPJwsl] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/AVHRR][LPJwsl]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][ORCHIDEE] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/AVHRR][ORCHIDEE]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][ORCHIDEE-CNP] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/AVHRR][ORCHIDEE-CNP]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][WorkConfront][LeafAreaIndex/MODIS][DLEM]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 282, in extractTimeSeries
    var = Variable(filename       = pathName,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Variable.py", line 132, in __init__
    out = il.FromNetCDF4(filename,variable_name,alternate_vars,t0,tf,group=groupname,convert_calendar=convert_calendar)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 790, in FromNetCDF4
    t,t_bnd,cbounds,begin,end,calendar = GetTime(var,t0=t0,tf=tf,convert_calendar=convert_calendar)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 246, in GetTime
    tb = CreateTimeBounds(t,alpha=GuessAlpha(t))
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 149, in GuessAlpha
    t0 = cf.num2date(t[0],t.units,calendar=t.calendar)
  File "src/cftime/_cftime.pyx", line 547, in cftime._cftime.num2date
  File "src/cftime/_cftime.pyx", line 103, in cftime._cftime._dateparse
ValueError: units must be one of 'seconds', 'minutes', 'hours' or 'days' (or singular version of these), got 'years'

[DEBUG][0][extractTimeSeries][JSBACH] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/MODIS][JSBACH]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][LPJwsl] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/MODIS][LPJwsl]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][ORCHIDEE] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/MODIS][ORCHIDEE]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][ORCHIDEE-CNP] Could not find [lai] in the model results
[DEBUG][0][WorkConfront][LeafAreaIndex/MODIS][ORCHIDEE-CNP]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][LPJ-GUESS] Could not find [rh] in the model results
[DEBUG][0][WorkConfront][NetEcosystemExchange/FLUXCOM][LPJ-GUESS]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][CombineVariables][MonotonicityError]
   0: t = [47465.500, 59844.500]  * /Volumes/Elements/ILAMB_analysis/MODELS/LPJwsl/npp/LPJ_S3_npp.nc
   1: t = [47465.500, 59844.500] ** /Volumes/Elements/ILAMB_analysis/MODELS/LPJwsl/npp/LPJ_S3_npp_edited.nc
[DEBUG][0][WorkConfront][NetEcosystemExchange/FLUXCOM][LPJwsl]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 328, in extractTimeSeries
    v = il.CombineVariables(V)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 1932, in CombineVariables
    raise MonotonicityError()
ILAMB.ilamblib.MonotonicityError: MonotonicityError

[DEBUG][0][extractTimeSeries][VISIT] Could not find [npp] in the model results
[DEBUG][0][WorkConfront][NetEcosystemExchange/FLUXCOM][VISIT]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][LPJ-GUESS] Could not find [rh] in the model results
[DEBUG][0][WorkConfront][NetEcosystemExchange/FLUXNET2015][LPJ-GUESS]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][CombineVariables][MonotonicityError]
   0: t = [51480.500, 60209.500]  * 
   1: t = [51480.500, 60209.500] ** 
[DEBUG][0][WorkConfront][NetEcosystemExchange/FLUXNET2015][LPJwsl]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 328, in extractTimeSeries
    v = il.CombineVariables(V)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ilamblib.py", line 1932, in CombineVariables
    raise MonotonicityError()
ILAMB.ilamblib.MonotonicityError: MonotonicityError

[DEBUG][0][extractTimeSeries][VISIT] Could not find [npp] in the model results
[DEBUG][0][WorkConfront][NetEcosystemExchange/FLUXNET2015][VISIT]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

[DEBUG][0][extractTimeSeries][LPJ-GUESS] Could not find [rh] in the model results
[DEBUG][0][WorkConfront][NetEcosystemExchange/GBAF][LPJ-GUESS]
Traceback (most recent call last):
  File "/opt/miniconda3/envs/ilamb/bin/ilamb-run", line 384, in WorkConfront
    c.confront(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 349, in confront
    obs,mod = self.stageData(m)
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/Confrontation.py", line 296, in stageData
    mod = m.extractTimeSeries(self.variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 316, in extractTimeSeries
    v = self.derivedVariable(variable,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 376, in derivedVariable
    Vs[arg] = self.extractTimeSeries(arg.name,
  File "/opt/miniconda3/envs/ilamb/lib/python3.9/site-packages/ILAMB/ModelResult.py", line 326, in extractTimeSeries
    raise il.VarNotInModel()
ILAMB.ilamblib.VarNotInModel: VarNotInModel

